W-18079979 Add missing button to troubleshooting topic (#823)
mulesoft/docs-gateway: gateway/1.9/modules/ROOT/pages/flex-troubleshoot-managed-creation.adoc

When creating and deploying a Managed Flex Gateway instance, you might encounter issues. Check these topics for possible causes and solutions.

> * xref:#add-button-missing[Add Managed Flex Gateway Button is Missing]

Reviewer: Those xrefs work? I've never seen them formatted this way before.

Author: They *do* work but apparently that's non-standard. Fixing.

* xref:#add-button-disabled[Add Managed Flex Gateway Button is Disabled]
* xref:#private-space[Private Space Not Available]

[[add-button-missing]]
== Add Managed Flex Gateway Button is Missing

The *Add Managed Flex Gateway* button in Runtime Manager might be missing because the selected business group does not have Managed Flex Gateway resources. Instead you see a "Start protecting your APIs with Managed Flex Gateways" message and a *Create a Self-Managed Flex Gateway* button.

> Check Business Group Managed Flex Gateway Resources::

Reviewer: ```suggestion
Check Your Business Group Managed Flex Gateway Resources::
```

> . To verify that the subscription model for your organization includes Managed Flex Gateway, navigate to *Access Management > Subscription > Runtime Manage Subscription*. If your subscription tier does not include *Small Managed Flex Gateways* and *Large Managed Flex Gateways* resources, contact your MuleSoft representative for more information.

Reviewer: ```suggestion
. Navigate to *Access Management > Subscription > Runtime Manage Subscription*. If your subscription tier doesn't include *Small Managed Flex Gateways* and *Large Managed Flex Gateways* resources, contact your MuleSoft representative for more information.

```

> . If your business group shows quantity *0/0 in use* or shows *Reassigned* for *Small Managed Flex Gateways* or *Large Managed Flex Gateways*, you may need to redestribute Managed Flex Gateway resources from one organization or business group to your selected business group. See xref:access-management::business-groups.adoc#redistribute-resources[Redistribute Resources Between Existing Business Groups].

Reviewer: ```suggestion
. If your business group shows quantity *0/0 in use* or shows *Reassigned* for *Small Managed Flex Gateways* or *Large Managed Flex Gateways*, try redestributing Managed Flex Gateway resources from one organization or business group to your selected business group. See xref:access-management::business-groups.adoc#redistribute-resources[Redistribute Resources Between Existing Business Groups].
```

Reviewer: To avoid "you may need to"
